bundler/lib/dependabot/bundler/update_checker/shared_bundler_helpers.rb

```diff
@@ -174,8 +174,11 @@ def inaccessible_git_dependencies
               }
             )
             git_specs.reject do |spec|
+              uri = URI.parse(spec.fetch("auth_uri"))
+              next false if uri.scheme != "http" && uri.scheme != "https"
```

Comment:
Should this be `||` instead of `&&`?

Reply:
I don't think so:
```
irb(main):001:0> u = URI.parse("https://github.com"); u.scheme != "http" && u.scheme != "https"
=> false
irb(main):002:0> u = URI.parse("https://github.com"); u.scheme != "http" || u.scheme != "https"
=> true
```

Maybe `next unless ["http","https"].include?(uri.scheme)` is easier to read? 
Or the [De Morgan](https://en.wikipedia.org/wiki/De_Morgan%27s_laws#Set_theory_and_Boolean_algebra), `next unless scheme == "http" || scheme =="https"`?